### PR TITLE
Ignore picking labels not currently visible

### PR DESCRIPTION
**Pull request by tallytalwar**

### CODE REVIEW

**Comment by tallytalwar** [comment]
@karimnaaji can you check this please, thanks.

File: core/src/labels/labels.cpp [hunk 1]
```diff
@@ -313,6 +313,8 @@ const std::vector<TouchItem>& Labels::getFeaturesAtPoint(const View& _view, floa
 
             for (auto& label : labelMesh->getLabels()) {
 
+                if (label->state() != Label::State::visible) { continue; }
```
Comment (karimnaaji) [inline core/src/labels/labels.cpp]: You can directly use https://github.com/tangrams/tangram-es/blob/master/core/src/labels/label.cpp#L169

Comment (tallytalwar) [inline core/src/labels/labels.cpp]: :+1:

**Comment by karimnaaji** [comment]
Also, this option should still be considered in the test https://github.com/tangrams/tangram-es/blob/pickFeature-fix/core/src/labels/labels.cpp#L289

**Comment by tallytalwar** [comment]
@karimnaaji thanks.

**Comment by karimnaaji** [comment]
Good to be merged!

**Pull request merged**